models/Customers.js

```diff
@@ -31,22 +31,24 @@ module.exports.getCustomer = (custID) => {
   });
 };
 
-module.exports.addCustomer = ({ first_name, last_name, join_date }) => {
+module.exports.addCustomer = ({ first_name, last_name, join_date, active }) => {
   return new Promise( (resolve, reject) => {
     db.run(`INSERT INTO customers 
-    VALUES(null, "${first_name}", "${last_name}", "${join_date}", 1 )`, function(err, computer) {
+    VALUES(null, "${first_name}", "${last_name}", "${join_date}", "${active}" )`, function(err, computer) {
       if (err) return reject(err);
       resolve({ id : this.lastID });
       }
     );
   });
 };
 
-module.exports.editCustomerName = ({ customer_id, first_name, last_name }) => {
+module.exports.editCustomerObj = ({ customer_id, first_name, last_name, join_date, active }) => {
   return new Promise( (resolve, reject) => {
     db.run(`UPDATE customers SET
     first_name="${first_name}",
-    last_name="${last_name}"
+    last_name="${last_name}",
+    join_date="${join_date}",
+    active="${active}"
     WHERE customer_id = ${customer_id}`, function(err, computer) {
       if (err) return reject(err);
       resolve({ id : this.changes });
```

Comment:
can you resolve the name of this property as "Number of changes made" or something other than "id"? Could be confusing to the developer